### PR TITLE
Support netstandard 2.0 directly

### LINKED ISSUE #82: Support netstandard2.0 directly
Split from #79

As @thomaslevesque notes, the [Cross-platform targeting](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting) advice from Microsoft says

> ✔️ DO start with including a netstandard2.0 target.
> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.

> ✔️ DO include a netstandard2.0 target if you require a netstandard1.x target.
> All platforms supporting .NET Standard 2.0 will use the netstandard2.0 target and benefit from having a smaller package graph while older platforms will still work and fall back to using the netstandard1.x target.

It is not clear that SelfInitializingFakes would benefit from supporting netstandard2.1 at this point .

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #82.

### CODE REVIEW

**Comment by blairconrad** [comment]
@thomaslevesque, assuming the build passes, wanna throw your 👀 on this? You're real good at the build/package shenanigans and might call me out for being too cutesy or something.
Oh! The build passed!

File: SelfInitializingFakes.sln [hunk 1]
```diff
@@ -32,11 +32,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SelfInitializingFakes.Tests
...
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SelfInitializingFakes.Tests.FIE.3.4.2", "tests\SelfInitializingFakes.Tests.FIE.3.4.2\SelfInitializingFakes.Tests.FIE.3.4.2.csproj", "{7F7EC34C-AA06-460B-B0B7-EF429DD99E21}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SelfInitializingFakes.Tests.FIE.4.9.2", "tests\SelfInitializingFakes.Tests.FIE.4.9.2\SelfInitializingFakes.Tests.FIE.4.9.2.csproj", "{FB3705B3-8BD8-4293-9266-6DE11FDACB2B}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SelfInitializingFakes.Tests.FIE.4.9.2", "tests\SelfInitializingFakes.Tests.FIE.4.9.2\SelfInitializingFakes.Tests.FIE.4.9.2.csproj", "{FB3705B3-8BD8-4293-9266-6DE11FDACB2B}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SelfInitializingFakes.Tests.FIE.5.5.0", "tests\SelfInitializingFakes.Tests.FIE.5.5.0\SelfInitializingFakes.Tests.FIE.5.5.0.csproj", "{27DE54D0-FB17-4A30-AB53-2FC98EEF2C4C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SelfInitializingFakes.Tests.FIE.5.5.0", "tests\SelfInitializingFakes.Tests.FIE.5.5.0\SelfInitializingFakes.Tests.FIE.5.5.0.csproj", "{27DE54D0-FB17-4A30-AB53-2FC98EEF2C4C}"
```
Comment (thomaslevesque) [inline SelfInitializingFakes.sln:39]: They *still* haven't fixed this in VS.. It's really annoying :/

**Review by thomaslevesque — APPROVED** [review]
Looks good to me @blairconrad, nice job!

File: SelfInitializingFakes.sln (hunk 1 above)
Comment (blairconrad) [reply SelfInitializingFakes.sln:39]: Can you explain? I get the feeling that maybe I fixed it. Or worked around it, but I am not exactly sure what _it_ is!

Comment (blairconrad) [reply SelfInitializingFakes.sln:39]: Ah. https://stackoverflow.com/questions/47312163/dotnet-sln-add-wrong-project-type-guid

**Comment by blairconrad** [comment]
Thanks for the review, @thomaslevesque!

File: SelfInitializingFakes.sln (hunk 1 above)
Comment (thomaslevesque) [reply SelfInitializingFakes.sln:39]: Yes. I've been annoyed by this [for a while](https://github.com/dotnet/project-system/issues/1821#issuecomment-416575784)...

**Pull request merged**